src/Form/FormControl.js

```diff
@@ -11,6 +11,7 @@ export const styleSheet = createStyleSheet('FormControl', () => {
       display: 'flex',
       flexDirection: 'column',
       position: 'relative',
+      transform: 'translateZ(0)', // #6076 prevents animation glitches of children to bubble up
```

Comment:
It promoting every `FormControl` into it's own layer. Not great. You can have a look at [The Anatomy of a Frame](https://aerotwist.com/blog/the-anatomy-of-a-frame/)  for some context.
I think that we should close this issue as most likely a Chrome issue.